lib/c-api/src/wasm_c_api/engine/config/jsc.rs

```diff
@@ -0,0 +1,38 @@
+use crate::{
+    error::update_last_error,
+    wasm_c_api::engine::{wasm_config_t, wasm_engine_t, wasmer_engine_t},
+};
+
+use super::wasmer_engine_config_t;
+
+/// Configuration specific for the `jsc` engine.
+///
+/// This is a Wasmer-specific type with Wasmer-specific functions for
+/// manipulating it.
+///
+/// cbindgen:ignore
+#[repr(C)]
+#[derive(Debug, Default)]
+pub struct wasmer_jsc_engine_config_t;
+
+/// Create a new  [`wasm_engine_t`] backed by a `jsc` engine.
+pub fn wasm_jsc_engine_new_with_config(config: wasm_config_t) -> Option<Box<wasm_engine_t>> {
```

Comment:
This should not be public

Reply:
[5c08932](https://github.com/wasmerio/wasmer/pull/5443/commits/5c08932b8602ade591afaabe379c52da6f8d57d4)